documentation/modules/proc-creating-configmap.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-external-logging.adoc
+
+[id='creating-configmap_{context}']
+= Creating a ConfigMap for logging
+
+To use a ConfigMap to define logging properties, you create the ConfigMap and then reference it as part of the logging definition in the `spec` of a resource.
+
+The ConfigMap must contain the appropriate logging configuration.
+
+* `log4j.properties` for Kafka components
```

Comment:
log4j is for the Kafka bridge as well.

Reply:
Yes. We should make it obvious, so changed to: 

* `log4j.properties` for Kafka components and the Kafka Bridge